### PR TITLE
ui: Tune badge colors

### PR DESCRIPTION
**Pull request by Etsija**
Screenshot from 2024-10-29 17-07-37

Please see the commits for details.

### CODE REVIEW

**Comment by sschuberth** [comment]
Can we get the usual nice screenshot? 😀

File: ui/src/routes/_layout/admin/runs/index.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ const columns = [
     header: 'Run Status',
     cell: ({ row }) => (
       <Badge
-        className={`border ${getStatusBackgroundColor(row.original.status)} hover:bg-${getStatusBackgroundColor(row.original.status)}`}
+        className={`border ${getStatusBackgroundColor(row.original.status)}`}
```
Comment (mmurto) [inline ui/src/routes/_layout/admin/runs/index.tsx:144]: So I would rather add `pointer-events-none` to these places where `Badge` is used and keep the colors as-is.

**Review by mmurto — CHANGES_REQUESTED** [review]

File: ui/src/routes/_layout/admin/runs/index.tsx (hunk 1 above)
Comment (Etsija) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: > So I would rather add `pointer-events-none` to these places where `Badge` is used and keep the colors as-is.

So remove them from get... color class helper functions and add them instead everywhere where a Badge is used?

Comment (mmurto) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: Exactly.

Comment (Etsija) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: > Exactly. 

So when you say "a bit more idiomatic" you really mean "blow up this tiny PR"? : 😏😏

Comment (mmurto) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: Sure you could include the `pointer-events-none` as a shared value in the helper function return if that makes more sense to you. The non-idiomatic part is specifying hover color when what we actually want is the color not to change.

Comment (mmurto) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: More options could be to create a wrapper component like `StatusBadge` that does it, or as it feels that we by-default don't want the badges to have hover color, just remove the hover colors from [here](https://github.com/eclipse-apoapsis/ort-server/blob/main/ui/src/components/ui/badge-variants.ts#L17-L23).

Comment (Etsija) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: I agree the last option would probably be best.

Comment (Etsija) [reply ui/src/routes/_layout/admin/runs/index.tsx:144]: Done and seems to be working.

**Review by mmurto — APPROVED** [review]

**Pull request merged**